src/main/java/pl/touk/sputnik/Test.java

```diff
@@ -5,10 +5,11 @@
     public boolean test = false;
 
     public int method() {
+        String str = null;
         if (test) {
             return 1;
         }
-        return 2;
+        return str.toUpperCase().length();
```

Comment:
[FindBugs] WARNING: NP: Null pointer dereference of str in pl.touk.sputnik.Test.method()